### PR TITLE
Add SR to mypy files

### PR DESCRIPTION
**Pull request by ThanatosGit**

### CODE REVIEW

File: randovania/games/samus_returns/exporter/game_exporter.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def _do_export_game(
         monitoring.set_tag("msr_target_platform", export_params.target_platform.value)
         monitoring.set_tag("msr_target_version", export_params.target_version.value)
 
-        from open_samus_returns_rando.version import version as open_samus_returns_rando_version
+        from open_samus_returns_rando.version import version as open_samus_returns_rando_version  # type: ignore
```
Comment (Miepee) [inline randovania/games/samus_returns/exporter/game_exporter.py:76]: I feel like if we're adding mypy, we might as well use typings from the patcher. Should just be adding one file somewhere.

Comment (ThanatosGit) [reply randovania/games/samus_returns/exporter/game_exporter.py:76]: Yes, but I won't update the patcher now for this.

Comment (Miepee) [reply randovania/games/samus_returns/exporter/game_exporter.py:76]: eh fair enough

**Review by Miepee — APPROVED** [review]

File: randovania/games/samus_returns/exporter/game_exporter.py (hunk 1 above)
Comment (ThanatosGit) [reply randovania/games/samus_returns/exporter/game_exporter.py:76]: I open an issue for it in the patcher repo

**Pull request merged**